tests/profiling/test_profiler.py

```diff
@@ -411,3 +411,15 @@ def test_profiler_serverless(monkeypatch):
     p = profiler.Profiler()
     assert isinstance(p._scheduler, scheduler.ServerlessScheduler)
     assert p.tags["functionname"] == "foobar"
+
+
+@pytest.mark.subprocess()
+def test_profiler_ddtrace_deprecation():
+    """
+    ddtrace interfaces loaded by the profiler can be marked deprecated, and we should update
+    them wen this happens.  As reported by https://github.com/DataDog/dd-trace-py/issues/8881
+    """
+    import warnings
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("error", DeprecationWarning)
```

Comment:
do we need to do something in this block?

I think `catch_warnings` only works for code executed within the block, so we might need to import the profiler or do something here?

Reply:
Haha, linter was helpfully removing unused imports.